stylesheets/page/cart-management.css

```diff
@@ -0,0 +1,60 @@
+@import url("/stylesheets/style.css");
+
+.gh-hr-card {
+  justify-content: space-between;
+  height: fit-content;
+  border-radius: 0.2rem;
+}
+
+.gh-cart-wrap {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
All the flex and its properties can be moved to utility classes of the component library.

Reply:
Will implement that.